jersey/src/main/java/com/netflix/conductor/server/common/BulkResponse.java

```diff
@@ -0,0 +1,71 @@
+package com.netflix.conductor.server.common;
+
+import org.apache.commons.lang3.tuple.MutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Synchronous response to return a list of failed and succeeded entities for the bulk request.
+ */
+public class BulkResponse {
+
+    private final List<String> bulkSuccessfulResults;
+    private final List<Pair<String,String>> bulkErrorResults;
```

Comment:
Add documentation about what `Pair<String, String>` correlate to.

Reply:
Why not return a `Map<String, String>` what does `List<Pair<String,String>>` provide ?